.eslintrc

```diff
@@ -4,8 +4,18 @@
     "browser": true,
     "jest": true
   },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module",
+    "ecmaFeatures": {
+      "jsx": true,
+      "modules": true
+    }
+  },
   "extends": "airbnb",
```

Comment:
you can add this plugins,
`  "plugins": [
      "react",
      "jsx-a11y",
      "import"
  ]`